sketch_map_tool/templates/digitize.html

```diff
@@ -48,6 +48,7 @@
             </div>
             <div>
                 <p>
+                    <input type="checkbox" id="consent" name="consent" value="True" checked>
                     {{ _('By uploading the file(s), you agree that the content of the files and the time of upload will
                     be saved and used for the further development of the service.') }}
```

Comment:
This text doesn't fit for a checkbox. It states that “if you upload the files, you consent”, not “by checking the box …”. Before merging this needs reviewing.